config/routes.rb

```diff
@@ -80,6 +89,7 @@
   get '/opportunities', to: 'pages#opportunities'
   get '/slack', to: 'pages#slack'
   get '/resources', to: 'pages#resources'
+  get '/videos', to: 'pages#video'
```

Comment:
No need for this either

Reply:
I don't understand why we didn't need this.